yarn-project/simulator/src/avm/avm_tree.ts

```diff
@@ -437,18 +516,14 @@ export class AvmEphemeralForest {
       else {
         lowPublicDataIndex = curr.getNextIndex();
         if (this.hasLocalUpdates(treeId, lowPublicDataIndex)) {
-          curr = this.getIndexedUpdates(treeId, lowPublicDataIndex)!;
+          curr = this.getIndexedUpdate(treeId, lowPublicDataIndex)!;
```

Comment:
when we enter this loop we probably need to track that `pathAbsentInEphemeralTree = false`